crates/sui-core/src/consensus_adapter.rs

```diff
@@ -1133,8 +1097,8 @@ mod adapter_tests {
     use super::position_submit_certificate;
     use crate::consensus_adapter::{
         ConnectionMonitorStatusForTests, ConsensusAdapter, ConsensusAdapterMetrics,
-        LazyNarwhalClient,
     };
+    use crate::mysticeti_adapter::LazyMysticetiClient;
```

Comment:
thoughts on renaming to `LazyConsensusClient`?

Reply:
Added a TODO.